engine/flink/kafka-components-utils/src/test/scala/pl/touk/nussknacker/engine/kafka/source/flink/KafkaSourceFactoryDefinitionExtractorSpec.scala

```diff
@@ -16,14 +16,14 @@ class KafkaSourceFactoryDefinitionExtractorSpec extends KafkaSourceFactoryProces
     // and they must not be returned by other services.
     extractedTypes should contain allOf (
       ClazzDefinition(Typed.genericTypeClass(classOf[SampleKey],Nil), Map(
-        "partOne" -> List(StaticMethodInfo(ParameterList(Nil, None), Typed[String], "partOne", None)),
-        "partTwo" -> List(StaticMethodInfo(ParameterList(Nil, None), Typed[Long], "partTwo", None)),
-        "toString" -> List(StaticMethodInfo(ParameterList(Nil, None), Typed[String], "toString", None))
+        "partOne" -> List(StaticMethodInfo(MethodTypeInfo(Nil, None, Typed[String]), "partOne", None)),
```

Comment:
maybe we should add factory methods like `MethodTypeInfo.noArg`, `MethodTypeInfo.withoutVarargs()`? This is is a part of API so it would be nice to have some helpers to speed up devs work